lib/ex_doc.ex

```diff
@@ -25,6 +25,7 @@ defmodule ExDoc do
       project: nil,
       retriever: ExDoc.Retriever,
       source_beam: nil,
```

Comment:
I have another PR that is waiting for this one to be merged, and it deals with normalizing options individually, I guess I can include this one rule